src/applications/ivc-champva/10-7959a/chapters/signerInformation.js

```diff
@@ -0,0 +1,138 @@
+import { cloneDeep } from 'lodash';
+import { VaTextInputField } from 'platform/forms-system/src/js/web-component-fields';
+import {
+  addressUI,
+  addressSchema,
+  fullNameUI,
+  fullNameSchema,
+  titleUI,
+  titleSchema,
+  radioUI,
+  radioSchema,
+  phoneUI,
+  phoneSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+const fullNameMiddleInitialUI = cloneDeep(fullNameUI());
+fullNameMiddleInitialUI.middle['ui:title'] = 'Middle initial';
+
+export const certifierRoleSchema = {
+  uiSchema: {
+    ...titleUI('Your information'),
+    certifierRole: radioUI({
+      title: 'Which of these best describes you?',
+      required: () => true,
+      labels: {
+        applicant: 'I’m the beneficiary submitting a claim for myself',
+        other:
+          'I’m a representative submitting a claim on behalf of the beneficiary',
+      },
+    }),
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      titleSchema,
+      certifierRole: radioSchema(['applicant', 'other']),
+    },
+  },
+};
+
+export const certifierNameSchema = {
+  uiSchema: {
+    ...titleUI('Your name'),
+    certifierName: fullNameMiddleInitialUI,
+  },
+  schema: {
+    type: 'object',
+    properties: {
+      titleSchema,
+      certifierName: fullNameSchema,
+    },
+  },
+};
+
+export const certifierAddressSchema = {
+  uiSchema: {
+    ...titleUI(
+      'Your mailing address',
+      'We’ll send any important information about this form to this address',
+    ),
+    certifierAddress: addressUI(),
+  },
+  schema: {
+    type: 'object',
+    required: ['certifierAddress'],
+    properties: {
+      titleSchema,
+      certifierAddress: addressSchema(),
+    },
+  },
+};
+
+export const certifierPhoneSchema = {
+  uiSchema: {
+    ...titleUI(
+      'Your contact information',
+      'We’ll use this information to contact you if we have more questions.',
+    ),
+    certifierPhone: phoneUI(),
+  },
+  schema: {
+    type: 'object',
+    required: ['certifierPhone'],
+    properties: {
+      titleSchema,
+      certifierPhone: phoneSchema,
+    },
+  },
+};
+
+export const certifierRelationshipSchema = {
```

Comment:
Relationship schema has an optional text field that only appears if the "other" radio option is selected.